src/sdk/accountspace_nb.js

```diff
@@ -124,14 +121,12 @@ class AccountSpaceNB {
         if (params.new_iam_path !== undefined) iam_path = params.new_iam_path;
         if (params.new_username !== undefined) user_name = params.new_username;
         const iam_arn = iam_utils.create_arn_for_user(requested_account._id.toString(), user_name, iam_path);
-        const new_account_name = new SensitiveString(`${params.new_username}:${requesting_account.name.unwrap()}`);
+        const new_account_name = account_util.get_account_name_from_username(params.new_username, requesting_account._id);
```

Comment:
Notice here that if only the `--new-path` and `params.new_username` changed the name and email will have `undefined` at the beginning.